dowhy/causal_estimator.py

```diff
@@ -62,6 +70,9 @@ def __init__(
         confidence_level: float = DEFAULT_CONFIDENCE_LEVEL,
         need_conditional_estimates: Union[bool, str] = "auto",
         num_quantiles_to_discretize_cont_cols: int = NUM_QUANTILES_TO_DISCRETIZE_CONT_COLS,
+        num_bootstrap_samples: int = DEFAULT_NUMBER_OF_BOOTSTRAP_SAMPLES,
+        seed: int = DEFAULT_BOOTSTRAP_SEED,
+        progressbarcolor: str = PROGRESS_BAR_COLOR,
```

Comment:
Should this really be a property of the estimator? Could we instead have a context manager that controls the default behaviour of progress bars?